arangod/RestServer/NonceFeature.cpp

```diff
@@ -44,11 +44,4 @@ void NonceFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
                              "the size of the hash array for nonces", true);
 }
 
-void NonceFeature::prepare() {
-  constexpr uint64_t initialSize = 2 * 1024 * 1024;
```

Comment:
Initialization of nonces now happens in `lib/Basics/Nonce.cpp` lazily, with the same `initialSize` as was used here before.